geth-utils/go.mod

```diff
@@ -6,3 +6,6 @@ require (
 	github.com/ethereum/go-ethereum v1.10.12
 	github.com/holiman/uint256 v1.2.0
 )
+
+// Uncomment for debugging
+// replace github.com/ethereum/go-ethereum => ../../go-ethereum
```

Comment:
Can we add this to the README or some other place so that for other people that is visible?

Reply:
I've added some lines to the `geth-utils` README :)